lib/player.js

```diff
@@ -5755,6 +5760,22 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return info;
   }
 
+  /**
+   * Get latency in milliseconds between the live edge and what's currently
+   * playing.
+   *
+   * @return {?number} The latency in milliseconds, or null if nothing
+   * is playing.
+   */
+  getLiveLatency() {
```

Comment:
If you intend this to be used by applications, please add `@export`.  Otherwise, it's fine as is.

Reply:
We created this to avoid code duplication, as the player calculates it, and we also needed it in the cmcd_manager. It’s not intended for application-level use.